features/sampling.feature

```diff
@@ -3,6 +3,29 @@ Feature: Sampling spans
   Background:
     Given I clear the Bugsnag cache
 
+  Scenario: Override sampling in config 1
+    Given I set the sampling probability for the next traces to "0"
```

Comment:
`0` should mean that no traces are received?

Reply:
The idea here is that even though i set the pvalue update request response to 0, i should still get traces because the probability is overriden to 1 in the config.

Reply:
Ah - I see now, thanks. The step is now confusingly worded now that we've added a "sampling probability" config option.

Can you compensate a little by enhancing the scenario text?

Maybe:

> Setting fixed sampling probability of 1 with dynamic probability of 0 should send all spans

Reply:
done 👍